gossip/gossip.go

```diff
@@ -700,9 +700,6 @@ func (g *Gossip) getNextBootstrapAddress() net.Addr {
 		g.resolverIdx %= len(g.resolvers)
 		g.resolversTried[g.resolverIdx] = struct{}{}
 		resolver := g.resolvers[g.resolverIdx]
-		if resolver.IsExhausted() {
```

Comment:
@petermattis @tamird recent changes seem to have broken this mechanism. The `IsExhausted()` check actually has value which over the course of the past two changes to this code has been first broken and now subtracted completely. Have we decided that we're never going to connect to nodes for gossip via a load balancer? If so, then this change makes sense. If not, then we're no longer going to be trying for additional connections through to find the sentinel, in the event we already have an active connection through the load balancer.